job_templates/service_action_-_enable_web_console.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Service Action - Enable Web Console
+job_category: Ansible Services
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
```

Comment:
Could you add `feature: enable_web_console` somewhere around here?

Reply:
Done.